root/lib/systemd/system/titus-metadata-proxy@.service

```diff
@@ -6,12 +6,16 @@ ConditionPathIsDirectory=/var/lib/titus-inits/%i/ns
 StartLimitIntervalSec=30
 StartLimitBurst=10
 
+After=titus-salt-generator.service
+Wants=titus-salt-generator.service
+
 [Service]
 Type=notify
 NotifyAccess=all
 WatchdogSec=30
 
 Environment=TITUS_PID_1_DIR=/var/lib/titus-inits/%i
+EnvironmentFile=/run/titus-metadata-proxy.salt
```

Comment:
We might want to make this a soft-dep, so:
`EnvironmentFile=-/run/titus-metadata-proxy.salt`

(If the dep file is failed to be written we'd rather be "available and insecure" than unavailable)